boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -204,23 +212,34 @@ Jumper settings for MEC1503 144WFBGA Socket DC Assy 6856 Rev B1p0
 =================================================================
 
 The MEC1503 ASSY 6856 CPU card does not include an onboard external
-32K crystal or oscillator. The one jumper block JP1 is for scoping
+32K crystal or oscillator. The one jumper block ``JP1`` is for scoping
 test clock outputs not for configuration. Please refer to schematic
 in reference section below.
 
 Programming and Debugging
 *************************
 
-This board comes with a Cortex ETM port which facilitates tracing and debugging
-using a single physical connection.  In addition, it comes with sockets for
-JTAG only sessions.
+Building
+==========
+
+#. Build :ref:`hello_world` application as you would normally do.
+
+#. Once you obtain ``zephyr.bin``, proceed to use the microchip tool `SPI Image Gen`_
+   in order to create the the final binary. The output generated by this tool is
+   what you need to flash in the SHD SPI memory.
```

Comment:
Could simplify the language a bit:

    Once you have ``zephyr.bin``, use the SPI Image Gen microchip tool to create the final binary.
    You need the output from this tool to flash ...

Reply:
done